src/index.jsx

```diff
@@ -11,6 +19,20 @@ root.render(
   </Provider>
 );
 
+store.subscribe(() => {
+  const { app } = store.getState();
+  renderAppLanguage(app);
+});
+
+// Update language in places that are out of the App's reach
+function renderAppLanguage({ language, languages }) {
+  const html = document.documentElement;
+  if (language && language !== html.lang) {
+    html.setAttribute("lang", language);
+    document.title = copy[language].page_title;
+  }
+}
+
```

Comment:
I don't want to put too much logic in `index.jsx`, but for updating the content of `index.html` it seems like the right place. 

Alternatively it can be thunked to an action handler. Which would be more correct as it only needs to run on `TOGGLE_LANGUAGE`, but also more complex.